src/progress-indicator/progress-indicator.component.ts

```diff
@@ -63,6 +63,29 @@ export class ProgressIndicator {
 	@Input() orientation: "horizontal" | "vertical" = "horizontal";
 	@Input() skeleton = false;
 
+	@Input() get current() {
+		return this.steps.indexOf(step => step.state[0] === "current");
+	}
+	set current(current: number) {
+		if (current === undefined) {
+			for (let i = 0; i < this.steps.length; i++) {
+				this.steps[i].state[0] = "incomplete";
+			}
+			return;
+		}
+
+		if (current > this.steps.length - 1) {
+			current = this.steps.length - 1;
+		}
```

Comment:
What if `current` is < 0?

We can't skip steps and complete them later? That was a requirement at some point.